README.md

```diff
@@ -268,6 +268,16 @@ Here is the list of well-known services that can be used without setting host an
 
 Then you need to implement `contact.html` template in your theme so that a contact form can be rendered at `/contact`.
 
+#### Disqus plugin
```

Comment:
Can we start having `README.md` within a plugin directory? See the example of `wp` plugin please.